zerocopy-derive/src/lib.rs

```diff
@@ -426,28 +426,7 @@ fn impl_block<D: DataExt>(
     // While there's no getting around this requirement for us, it does have
```

Comment:
Now that there's only one item in this list, can you make it not a list? Something like:

```rust
    // While there's no getting around this requirement for us, it does have the
    // pretty serious downside that, when lifetimes are involved, the trait
    // solver ties itself in knots:
```